### PR TITLE
Refactor Quick Start - Shift JDK17 Instructions to FAQ and Update Code Formatting for PDF Compatibility

### LINKED ISSUE #253: Shift JDK17 installation instructions to FAQ
This content is more relevant for the FAQ section

### PR DESCRIPTION
**Pull request by TechRaj**
Fixes #253 and #257 

This PR refines the **Quick Start** guide to improve readability and user experience:

1. **JDK17 Installation Instructions Moved to FAQ**:
   - The detailed JDK17 installation steps for Windows, macOS, and Ubuntu/Debian have been relocated to the FAQ section. This keeps the Quick Start guide streamlined for users who already have Java 17 installed.
   - Added a reference link in the Quick Start guide to direct users to the FAQ if they need installation instructions.

2. **Refactored Long Code Blocks Using Block Quotes**:
   - Updated long code blocks to use block quotes with line breaks, ensuring commands fit without horizontal scrolling.
   - This change improves readability in both Markdown and PDF formats, making commands easier to follow and copy.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -73,75 +73,34 @@ The following annotated text-boxes are used throught this guide to provide usefu
 
 ## Quick Start
 
-In this section, lets learn how to set up MediBase3 on your computer.
+In this section, let's learn how to set up MediBase3 on your computer.
 
 ### Installing MediBase3
 
-#### Step 1: Install JDK 17
+#### Step 1: Check if you have Java 17 installed your system.
```
Comment (PresleyChew) [inline docs/UserGuide.md:80]: "Check if you have Java 17 installed on your system." Missing "on"

**Review by PresleyChew — CHANGES_REQUESTED** [review]
Overall LGTM, ensure typo is fixed is ready for merge

File: docs/UserGuide.md (hunk 1 above)
Comment (yadobler) [reply docs/UserGuide.md:80]: **Java17**:
image

**Comment by TechRaj** [comment]
This by extension fixes #264

**Review by yong-zhuo — APPROVED** [review]

**Pull request merged**